spec/dummy/db/schema.rb

```diff
@@ -41,6 +41,8 @@
     t.string   "featured_image_content_type"
     t.integer  "featured_image_file_size"
     t.datetime "featured_image_updated_at"
+    t.string   "seo_title"
```

Comment:
Unnecessary spacing detected.<br>Put one space between the method name and the first argument.